macros/calibrateCTSEvents.C

```diff
@@ -0,0 +1,313 @@
+  float y0_layer2[33]={1.96, 1.95, 3.23, 1.97, 3.27, 2.77, 2.2, 2.14, 2.9, 3.05, 2.63, 1.54, 3.13, 1.32, 2.99, 2.48, 2.09, 2.1, 2.35, 3.22, 3.16, 2.48, 4.18, 2.86, 2.78, 2.27, 3.17, 1.9, 3.18, 2.0, 3.11, 3.78};
+  float m_layer3[33]={1.25, 0.66, 1.23, 1.25, 1.17, 1.34, 1.26, 1.17, 1.14, 1.15, 1.2, 1.22, 1.19, 1.22, 1.16, 1.24, 1.22, 1.27, 1.33, 1.31, 1.29, 0.36, 1.24, 1.23, 1.23, 1.25, 1.22, 1.26, 1.15, 1.26, 1.24, 1.17};
+  float y0_layer3[33]={-3.89, 2.46, -1.88, -4.09, -2.92, -6.09, -5.11, -1.42, -0.59, -2.47, -3.89, -3.66, -3.03, -4.23, -3.34, -4.73, -5.15, -5.66, -7.47, -6.34, -6.16, 14.12, -5.35, -5.09, -3.56, -3.58, -3.05, -4.27, -2.02, -3.65, -4.7, -2.97};
+  float m_layer4[33]={1.14, 1.21, 1.15, 1.19, 1.09, 1.12, 1.08, 1.15, 1.16, 1.07, 1.05, 1.12, 1.13, 1.11, 1.09, 1.03, 1.22, 1.17, 1.19, 1.2, 1.19, 1.18, 1.14, 1.15, 1.15, 1.15, 1.14, 1.05, 1.13, 1.19, 1.16, 1.04};
+  float y0_layer4[33]={-1.72, -5.08, -0.94, -3.44, -0.9, -3.02, -3.14, -3.65, -2.13, -1.14, 0.34, -1.86, -2.78, -2.01, -1.03, -1.16, -5.06, -3.55, -3.31, -4.24, -3.01, -3.39, -1.93, -2.53, -1.21, -4.04, -1.84, 0.15, -0.92, -3.89, -3.16, 0.22};
+  
+
+
+  // The loop
+  for (ULong_t entry = 0; entry < nSignals; entry++) {
+    if ((((entry+1)%100000) == 0) || (entry == (nSignals-1))) {
+      printf("\rprocessing signal %lu...", entry+1);
+      fflush(stdout);
+      std::cout<<std::setw(5)<<std::setiosflags(std::ios::fixed)<<std::setprecision(1)<<" "<<(100.*(entry+1))/nSignals<<" % done\r"<<std::flush;
+    }
+
+    signals->GetEntry(entry);
+
+    if ((ULong_t(eventNr) != prevEventNr) && (prevEventNr !=1)) {
+      module.removeEmpty();
+      event->setModule(module);
+      tree->Fill();
+
+      module.reset();
+    }
+   
+    timeCalib = (timeStamp-refTime)*1e9-time_calibration[int(layer)][int(chID)];
+    if(int(chID)==1) { ch1time = timeCalib; }
+    //printf("Layer:%g, ChID:%g,\t Timestamp:%1.15g,\t RefTime:%1.15g,\t timediff:%1.15g\t, timeToCh1:%1.15g\n",layer,chID,timeStamp,refTime,timeCalib,timeCalib-ch1time);
+    ToT = ToT*1e9;
+
+    //-----> Make channel mapping!
+    if (int(layer) == 1){  
+      switch(int(x)) {
+        case 1: totCalib=ToT*m_layer1[16]+y0_layer1[16]; break;
+        case 2: totCalib=ToT*m_layer1[15]+y0_layer1[15]; break;
+        case 3: totCalib=ToT*m_layer1[17]+y0_layer1[17]; break;
+        case 4: totCalib=ToT*m_layer1[14]+y0_layer1[14]; break;
+        case 5: totCalib=ToT*m_layer1[18]+y0_layer1[18]; break;
+        case 6: totCalib=ToT*m_layer1[13]+y0_layer1[13]; break;
+        case 7: totCalib=ToT*m_layer1[19]+y0_layer1[19]; break;
+        case 8: totCalib=ToT*m_layer1[12]+y0_layer1[12]; break;
+        case 9: totCalib=ToT*m_layer1[20]+y0_layer1[20]; break;
+        case 10: totCalib=ToT*m_layer1[11]+y0_layer1[11]; break;
```

Comment:
Adding to readability but absolutely low priority:

align all `totCalib`, make spaces before and after `=`, `*`, and `+` and align all `break`.